invenio_opendata/base/recordext/fields/opendata.cfg

```diff
@@ -24,7 +32,11 @@ _first_author, first_author, creator:
 
 _additional_authors, additional_authors, contributor:
     schema:
-        {'_additional_authors': {'type': 'list', 'force': True}}
+        {'_additional_authors': {'required': False, 
+        'schema' = {'a_dict': {'type': 'dict', 
```

Comment:
Note that bits like 'a_dict' would not be valid.

In view of Invenio master branch changes in favour of JSON Schema, let's close this WIP and let's address the data model configuration via future JSON Schema later.